_includes/configuration/extended-valid-elements.md

```diff
@@ -25,6 +25,12 @@ Also see [valid_elements]({{ site.baseurl }}/configure/content-filtering/#valid_
 
 To allow script elements in the editor, include the following in the {{site.productname}} configuration:
 
-```js
-extended_valid_elements : "script[src|async|defer|type|charset]"
-```
\ No newline at end of file
+```
+extended_valid_elements : 'script[src|async|defer|type|charset]'
+```
+
+## Live Example
+
+This example shows you how to use the [extended_valid_elements]({{ site.baseurl }}/configure/content-filtering/#extended_valid_elements) option. This option enables you to specify valid elements and attributes.
```

Comment:
Maybe `specify additional valid elements and attributes`? As it's appending to what's valid.

Reply:
Maybe fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb